configure.sh

```diff
@@ -441,7 +441,7 @@ while getopts "bBcdDEfgGhHk:MNO:PST-" opt; do
 
 done
 
-if [ "$bc_only" -eq 1 -a "$dc_only" -eq 1 ]; then
+if [ "$bc_only" -eq 1 ] && [ "$dc_only" -eq 1 ]; then
 	usage "Can only specify one of -b(-D) or -d(-B)"
```

Comment:
Why change all of the `-a` uses to `&&`?

Reply:
Not sure of the motivation in the context of this pull request, but `-a` and `-o` are XSI extensions marked as obsolescent:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html#tag_20_128_16

Reply:
Ah, yes, that is a sound reason.